### PR TITLE
Dicts iterate differently on python 2 vs. 3. Let's sort.

### PR DESCRIPTION
**Pull request by dlc37**
Inspired by python3 test failures before @Roguelazer modernized them. The order of 'all()' changes.

### CODE REVIEW

**Comment by Roguelazer** [comment]
I actually manually added the `message` field to our Exception subclass in #50, since otherwise the APIs were incompatible between Python 2 and Python 3.

I'm fine with this if tests pass, though.

**Comment by dlc37** [comment]
One of the tests failed; I've fixed the test. Not sure if Travis is rerunning the tests automagically...

**Comment by dlc37** [comment]
Tests pass.

File: tests/test_webhook.py [hunk 1]
```diff
@@ -28,7 +28,11 @@ def test_webhooks(per_run_unique):
 
     # Index webhooks
     webhooks = easypost.Webhook.all(url=expected_url)
-    assert webhooks["webhooks"][len(webhooks["webhooks"]) - 1].id == webhook.id
+    for wh in webhooks["webhooks"]:
```
Comment (Roguelazer) [inline tests/test_webhook.py]: how about just

`assert any(wh.id == webhook.id for wh in webhooks['webhooks'])`

Comment (dlc37) [inline tests/test_webhook.py]: Done, thanks.

File: tests/test_report.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ def test_shipment_report():
     )
 
     assert report.object == "ShipmentReport"
-    assert report.status == "available"
+    assert report.status in ("available", "new")
```
Comment (dlc37) [inline tests/test_report.py]: Do you think this is sufficient, or will the retrieve() call fail for new reports? I can't reproduce new reports.

Comment (Roguelazer) [inline tests/test_report.py]: I think it's sufficient. If not, we'll fix it next time it breaks.

**Comment by Roguelazer** [comment]
🛳it

**Pull request merged**